spec/duplicates_spec.rb

```diff
@@ -1,3 +1,4 @@
+require File.expand_path('../spec_helper', __FILE__)
```

Comment:
Just the old way of getting relative paths included. Since we're not supporting ruby 1.8 I'll change it to `require_relative`. 

http://stackoverflow.com/questions/4325759/is-require-file-expand-path-file-the-best-practice